api/src/wfl/module/covid.clj

```diff
@@ -185,11 +190,20 @@
         _       (jdbc/db-do-commands tx [(format create details)
                                          (format alter details)])
         items   (-> (select-keys request (keys tdr-source-serialized-fields))
+                    (update :dataset pr-str)
                     (set/rename-keys tdr-source-serialized-fields)
                     (assoc :details details)
                     (->> (jdbc/insert! tx tdr-source-table)))]
     [tdr-source-type (-> items first :id str)]))
 
+(defn ^:private load-tdr-source [tx {:keys [source_items] :as details}]
```

Comment:
Maybe it's just me, but I kept getting confused by `{:keys [source_items] :as details}`, and its "read stringified id" nature, it seems this map is not the same as our `source details` table, nor the source table, and to be able to use this, as we did in `update-tdr-source`, we had to make up a map: `(load-tdr-source tx {:source_items (str (:id source))})`. Would be helpful to add a docstring to this function!

Reply:
Actually NVM, after taking a look at the liquibase files, I realized `{:keys [source_items] :as details}` IS the `ContinuousWorkload` table. I have updated the lucidchart from `source_item` to `source_items` to match what we really have in DB now. In the meantime, should it be `{:keys [source_items] :as continuous_workload}` then?

Reply:
maybe `workload` - yeah. I'll change it!